src/docs/product/accounts/pricing.mdx

```diff
@@ -103,27 +74,73 @@ We have three tables for [errors](#errors-pricing), [transactions](#transactions
+If the amount of prepaid data you want to buy falls in-between increments, you may want to consider setting an on-demand budget, (which is based on a dollar amount rather than data volume), instead. 
+
+Use the slider on the [pricing](https://sentry.io/pricing/) page to determine the amount of prepaid data you might need based on the event types you send. 
+
+![Reserved Volume settings in Subscriptions](reserved-volume.png)
+
+For more information on how to manage your monthly spending, check out our [Quota Management documentation](/product/accounts/quotas/).
+
+## Volume {#capacity}
+
+Your volume is the total number of events and attachments — or the total amount of data— sent by your organization. If you'd like to purchase more than what's included in your [plan](https://sentry.io/pricing/), you can do so in two ways: by buying a **prepaid (or reserved) volume** and/or by setting up an **on-demand volume budget**, which will cover potential overages.
+
+### Prepaid (Reserved) Volume {#prepaid-reserved-capacity}
+
+When you buy reserved volume (or quota), you prepay for data at significant discounts. Your prepaid, reserved volume is always applied before on-demand volume kicks in. Unused reserved volume expires at the end of each billing month.
+
+Each Sentry plan includes different amounts of reserved volume. For example, our Team plan includes a minimum of 50k errors, 100k transactions, 500 replays, and 1GB of attachments per month. To see prepaid pricing for each plan, check out our [per-data-type-pricing](#per-data-type-pricing).
+
+
+
+
+
+### On-Demand Volume {#on-demand-capacity}
+
+Buying on-demand volume lets you control the maximum amount you’ll pay for overage each month after your reserved volume runs out. If you go over your on-demand volume,  you’ll still have access to previous event data, but any new data you send will be rejected and you won’t be charged for it.
+
+With on-demand volume, you pay for any event or attachment processed, (rounded up to the nearest cent), after your reserved volume has run out. On-demand volume is billed at the end of each billing month. To see on-demand pricing per plan type, check out our [per-data-type-pricing](#per-data-type-pricing).
+
+#### Shared and Per-Category On-Demand {#on-demand-cap}
+
+Organizations on all plans have the option to set a shared on-demand strategy. Organizations on plans with [performance monitoring](/product/performance/) features have the ability to choose between a shared or per-category on-demand strategy:
+
+- **Shared:** Your on-demand budget is shared between errors, transactions, replays and attachments.
+
+- **Per-category:** Separate on-demand budgets for errors, transactions, replays and attachments. Any overages in one category will not consume the budget of another category.
```

Comment:
```suggestion
- **Shared:** Your on-demand budget is shared between errors, transactions, replays and attachments.

- **Per-category:** Your on-demand budgets for errors, transactions, replays, and attachments are set up individually so that any overages in one category won't consume the budget of another category.
```